watchfaces/008-contemporary-digital-rings.qml

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
The complexity of this watchface makes it so that it makes the entire system UI lag.
On an emulator, I can see the asteroid-launcher blocked (for a quarter of a second) every time the frame is refreshed (every second). Maybe you could play with the "renderStrategy: Canvas.Threaded" http://doc.qt.io/qt-5/qml-qtquick-canvas.html#renderStrategy-prop to let the rendering happen in a thread and not block the asteroid-launcher rendering

Reply:
added renderStrategy: Canvas.Threaded before renderTarget and it gives and obvious increase in speed when switch to and away from the watchface. could not check in emulator though.

PR incoming

Reply:
If it works well. Maybe we should use that in the other complex watchfaces

Reply:
This suggestion made some heavy testing necessary.
Now, all canvases in all watchfaces run threaded but i had many awkward occurences while testing. When i added or left renderTarget: FramebufferObject in the canvases the watchfaces crashed the ui completely or the shadows where not rendered but displayed as black boxes.
After hours of fiddling i found that solely calling renderStrategy: Canvas.Threaded in all instances does the trick.
One obvious gain is that switching to and away from the watchfaces and selecting them in settings now feels more fluid imo.
Also some lags i did not relate to watchfaces as of now are gone on my bass! Like flickering and unreliable display when selecting watchfaces in settings or not showing wallpapers when testing many in a row.
The settings-watchfaces page diplays faster imo but all watchfaces somehow are a black box now for a split second when loading in settings page.
Please check for regressions related to this.
-> Changed all instances of 'renderTarget: Canvas.FramebufferObject' to 'renderStrategy: Canvas.Threaded' in all 11 watchfaces.